src/Controllers/SubmitControllerCreate.php

```diff
@@ -88,4 +117,47 @@ function ($value)
 
 		return true;
 	}
+
+	/**
+	 * Check the CMS Version
+	 *
+	 * @return  false on failiure else the CMS Version
+	 *
+	 * @since   1.0
+	 */
+	private function checkCMSVersion($data)
+	{
+		$dotchecks = explode('.' $data);
+
+		// We ever use 2 dots and 3 parts in our CMS Version
+		if (count($dotchecks) != 3)
+		{
+			return false;
+		}
+
+		// The pulugin is installed since 3.5.0 other CMS Versions can't have this plugin installed
+		if (version_compare($data, '3.5.0', '<'))
```

Comment:
We are now open for 3.0.0 - 4.0.0: https://github.com/joomla-extensions/jstats-server/commit/9208d2f0a5403990c1c4c94c56ed42763b8395b8